packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -1,86 +1,194 @@
+  const repoDomain = new URL(apiEndpoint).hostname.replace(".cdn", "");
+  const repoAddress = apiEndpoint.replace(".cdn.", ".").replace("/api/v2", "");
+  return (
+    <SideNav>
+      <SideNavLogo
+        repoName={repoName}
+        repoDomain={repoDomain}
+        href={repoAddress}
+      />
+
+      <SideNavUnorderedListTop>
+        <SideNavListItem>
+          <SideNavLink
+            title="Page types"
+            href="/"
+            active={
+              currentPath === "/" || currentPath.startsWith("/page-types")
+            }
+            onClick={handleNavigation}
+            Icon={(props) => <CUSTOM_TYPES_CONFIG.page.Icon {...props} />}
+          />
+        </SideNavListItem>
+
+        <SideNavListItem>
+          <SideNavLink
+            title="Custom types"
+            href="/custom-types"
+            active={currentPath.startsWith("/custom-types")}
+            onClick={handleNavigation}
+            Icon={(props) => <CUSTOM_TYPES_CONFIG.custom.Icon {...props} />}
+          />
+        </SideNavListItem>
+
+        <NavSeparator />
+
+        <SideNavListItem>
+          <SideNavLink
+            title="Changes"
+            href="/changes"
+            active={currentPath.startsWith("/changes")}
+            onClick={handleNavigation}
+            Icon={(props) => <RadarIcon {...props} />}
+            RightElement={
+              displayNumberOfChanges && (
+                <ChangesIndicator
+                  numberOfChanges={numberOfChanges}
+                  active={currentPath.startsWith("/changes")}
+                />
+              )
+            }
+          />
+        </SideNavListItem>
+
+        <NavSeparator />
+
+        {/**TODO: update this once we figure out how to display multiple slices libraries, make it it's own list with a header <SideNavListTitle>Slice Library</SideNavListTitle> */}
+
+        <SideNavListItem>
+          <SideNavLink
+            title="Slices"
+            href="/slices"
+            Icon={(props) => <FolderIcon {...props} />}
+            active={currentPath.startsWith("/slices")}
+            onClick={handleNavigation}
+          />
+        </SideNavListItem>
+      </SideNavUnorderedListTop>
+
+      {changelog.updateAvailable ||
+        (true && (
+          <UpdateInfo
+            href="/changelog"
+            onClick={handleChangeLogNavigationFromUpdateBox}
+          />
+        ))}
+
+      <SideNavUnorderedListBottom>
+        <VideoItem
+          id="video-tool-tip"
+          videoUrl={VIDEO_YOUTUBE_PLAYLIST_LINK}
+          onClose={() => {
+            setSeenTutorialsToolTip();
+          }}
+          onClick={(event) => {
+            setSeenTutorialsToolTip();
+            const maybeHref = event.currentTarget.getAttribute("href");
+            const href =
+              maybeHref !== null ? maybeHref : VIDEO_YOUTUBE_PLAYLIST_LINK;
+            void telemetry.track({
+              event: "open-video-tutorials",
+              video: href,
```

Comment:
❓Why not directly setting the video

```suggestion
              video: VIDEO_YOUTUBE_PLAYLIST_LINK,
```

Reply:
done